autotest.pm

```diff
@@ -265,8 +266,11 @@ sub start_process {
             $SIG{HUP}  = 'DEFAULT';
             $SIG{CHLD} = 'DEFAULT';
 
+            my $signal_blocker = signalblocker->new;
```

Comment:
Wasn't there a pattern to wrap this in a block instead of the explicit `undef`? Which would also ensure it's not used afterwards :thinking: